scripts/release/README.md

```diff
@@ -46,20 +50,55 @@ This section briefly describes the expected outcomes of the current build pipeli
+    1. Docker containers will package `deb` and `rpm` artifacts inside of Ubuntu 18.04 and Centos 7, respectively.
+
+    1. Jenkins will then pause to wait for [the only manual part of the build/package/test phase], which is to forward the `gpg-agent` that establishes a direct between the local machine that contains the signing keys and the remote ec2 instance.
+
+    1. Once the signatures have been verified, the all the build artificats (tarballs, `rpm` and `deb` packages, signatures) to an `s3` bucket.  Included in the bucket are the build logs.
 
 1. test
 
-    Download the `deb` and `rpm` packages from staging and test.
+    Download the `deb` and `rpm` packages from staging and test and locally.
+
+    This will spin up a local python server that will host both an `APT` repository and a `YUM` repository in Ubuntu 18.04 and Centos 7 docker containers, respectively.
+
+    Each package is downloaded in the respective environment and the following tests are done:
+
+    - The signatures are correct.
+    - The packages are built from the correct branch and channel and are the correct version.  This done by running `algod -v`.
+        + This is done for the following docker containers:
+            - centos:7
+            - fedora:28
+            - ubuntu:16.04
+            - ubuntu:18.04
+
+    - Creates a test network using `goal`.
+    - et. al.
+
+    > Centos 7 needs the GPG 2.2.9 version which is downloaded from `s3://algorand-devops-misc/tools/`.
 
 1. prod
 
-    Copy the build artifacts and their detached signatures from `algorand-builds` to the production `algorand-dev-deb-repo` bucket.  The [releases page] links to this location.
+    Copy the build artifacts and their detached signatures from the staging bucket to the production `algorand-dev-deb-repo` bucket.  The [releases page] links to this location.
+
+    This is done via the AWS cli tool (`awscli`):
+
+        `aws s3 sync s3://{SOURCE} s3://{DESTINATION}`
+
+    In addition, the releases page (located on s3 in the root of the `algorand-releases` bucket), is updated automatically by [running a python script] (although, this does need some more work to be fully automated).
+
+    > Note that the [`reverse_hex_timestamp`] script is used to create a bucket name so that it will always be the first available folder in the `algorand-dev-deb-repo` bucket.  This is used by the just-mentioned python release page script to ensure that it always grabs the most-recent build.
+    >
+    > For example:
+    >
+    >   `fa16c3cb5_2.0.13`
 
     In addition, local snapshots are used by Debian-based (`aptly`) and RHEL-based tooling to deploy the respective packages to `algorand-releases`.  These are the packages which can then by downloaded by `apt` and `yum`.
 
+    There are now three separate distributions: `stable`, `beta` and `nightly`.  All three distros contain multiple versions of the following packages, respectively: `algorand`, `algorand-beta` and `algorand-nightly`.
```

Comment:
Does this mean we can have a 'nightly' distribution with an algorand-beta package? Is there a reason we'd want this?

Reply:
It wouldn't be from the beta channel, at least I don't think so.  I don't think that the process for issuing nightly builds has been fully fleshed out yet, so I'll remove the bit about the nightly for now.